### PR TITLE
Fix Course Admin Area Filter Dropdown Values

### LINKED ISSUE #269: Update course admin area filter dropdown to get data from metadata state implemented as part of #229
Dependant on the merging of #224 and #229 into dev

### PR DESCRIPTION
**Pull request by natalynyu**
This PR updates the area filter dropdown in the Course Admin tab to use `areas` from the `metadata`. The column header was originally "Course Prefix," but Jonathan Seitz agreed that we could change the column header name to "Area."

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality
      to change)

## Checklist:

- [x] I have run `eslint` on the code
- [x] I have added JSDoc for all of my code (where applicable)
- [x] I have added tests to cover my changes.

## Priority:
- [x] Normal 
- [ ] High 

## Related Issues:

Fixes #269

### CODE REVIEW

**Review by rmainseas — CHANGES_REQUESTED** [review]
This is looking great. Just one small thing.

File: src/client/components/layout/__tests__/AppRouter.test.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ describe('App Router', function () {
     context('/course-admin', function () {
       it('renders the CourseAdmin component', async function () {
         const { findByText } = renderWithUser('/course-admin');
-        return findByText(/Course Prefix/);
+        return findByText(/Area/);
```
[the remark this answers is not in the document]
Comment (natalynyu) [reply src/client/components/layout/__tests__/AppRouter.test.tsx:50]: @rmainseas That's a good point. I was unsure of what to test for here to make sure that the CourseAdmin tab was unique to that page. From the other tests, it looks like we were trying to find the unique column name that only existed on that page and not other pages, but I'm not sure there's something unique I can test for for this tab.
image

[the remark this answers is not in the document]
Comment (rmainseas) [reply src/client/components/layout/__tests__/AppRouter.test.tsx:50]: Hmm. That's a good point. Perhaps `getByTestId` might help you here

[the remark this answers is not in the document]
Comment (natalynyu) [reply src/client/components/layout/__tests__/AppRouter.test.tsx]: Thanks Robert! I added a test id to Course Admin and updated the test to reflect this.

**Review by rmainseas — APPROVED** [review]
This looks great, merge away

**Pull request merged**